src/health.css

```diff
@@ -0,0 +1,75 @@
+* {
```

Comment:
this should be in global.css (same as 'body' styling). you can cange background by changing body inside health module

css ONLY for health module please put in src/modules/health